cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -850,7 +857,7 @@ func printShardStats(fn string) error {
 }
 
 func srcLogLevelIsDebug() bool {
-	lvl := os.Getenv("SRC_LOG_LEVEL")
+	lvl := os.Getenv(srcLogLevel)
```

Comment:
You can use the constant from the package: https://pkg.go.dev/github.com/sourcegraph/log#pkg-constants

```suggestion
	lvl := os.Getenv(log.EnvLogLevel)
```

Reply:
I see, we can probably use this constant for all references to the env var. I'll push a change for that reference tomorrow